packages/clerk-js/src/core/tokenCache.ts

```diff
@@ -73,16 +77,11 @@ const MemoryTokenCache = (prefix = KEY_PREFIX): TokenCache => {
 
     entry.tokenResolver
       .then(newToken => {
-        const nowSeconds = Math.floor(Date.now() / 1000);
         const expiresAt = newToken.jwt.claims.exp;
-        const expiresIn = expiresAt - nowSeconds;
-
-        if (expiresIn <= 0) {
-          clerkCoreErrorExpiredToken(expiresAt);
-        }
+        const issuedAt = newToken.jwt.claims.iat;
+        const expiresIn = expiresAt - issuedAt;
```

Comment:
```suggestion
        const expiresInSeconds = expiresAt - issuedAt;
```
or
```suggestion
        const expiresIn: Seconds = expiresAt - issuedAt;
```
or simply ignore as this is already fine :)